test/components/converters/test_docx_file_to_document.py

```diff
@@ -1,11 +1,12 @@
-import logging
 import json
-
+import logging
```

Comment:
We should be using the logger from haystack right?

Reply:
This is just reshuffled LOC, we used it before and use it throughout tests. Here perhaps because of:
```
     with caplog.at_level(logging.WARNING):
            docx_converter.run(sources=paths)
            assert "Could not read non_existing_file.docx" in caplog.text
```

Reply:
@sjrl 's point still stands though - we should import `logging` from `haystack` and replace format strings with interpolated ones to support our structured logger.

Reply:
I fully agree but here, in this test, logging is used just to lookup warning level for caplog. I'll replace it with a constant.

Reply:
Ah, sorry - didn't realize this was a comment on one of the tests 🤦